### PR TITLE
BlockShark Pledge

### PR DESCRIPTION
**Pull request by nad010286**

### CODE REVIEW

File: BlockShark/README.md
Comment (dckc) [inline BlockShark/README.md:3]:
```suggestion
To receive and maintain delegation from the Agoric OpCo, I, BlockShark hereby commit to the following work and support for the benefit of the Agoric Network and Community. If, for any reason, I do not meet these commitments then I understand that I may lose delegation staked to me by the Agoric Operations Company and/or any other associated organizations. :
```

**Review by dckc — COMMENTED** [review]
looks OK, once the entity name is filled in

@jeetraut the runbook links we generated are again not quite right. I'll let you decide whether to require that those are fixed before merging.

File: BlockShark/README.md
Comment (nad010286) [reply BlockShark/README.md]: added BlockShark

**Review by dckc — APPROVED** [review]

**Review by jeetraut — APPROVED** [review]
Looks good to me!

**Pull request merged**
